Testing/functional/e2e/cypress/integration/dependent/dependents.js

```diff
@@ -4,6 +4,7 @@ describe('dependents', () => {
     const firstName = "Sam"
     const lastName = "Testfive"
     const doB = "2014-03-15"
+    const invalidDoB = "2007-08-05"
```

Comment:
It is probably better to calculate this invalidDoB date based on Today date (ie. Today.substract(12 or 13).